build_scripts/windows/Product.wxs

```diff
@@ -3,7 +3,7 @@
 
   <?define ProductVersion="$(env.CLI_VERSION)" ?>
 
-  <?define ProductName = "Microsoft CLI 2.0 for Azure" ?>
+  <?define ProductName = "Microsoft Azure CLI" ?>
```

Comment:
This change makes the doc https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli#uninstall and  https://github.com/MicrosoftDocs/azure-docs-cli/pull/1440 no longer valid

> Once on this screen type __Azure CLI__ into the program search bar. The program to uninstall is listed as __Microsoft CLI 2.0 for Azure__. Select this application, then click the `Uninstall` button.